test/routable_component_routes.js

```diff
@@ -96,29 +96,29 @@ describe('RoutableComponentRoutes', function() {
   });
   /** @test {RoutableComponentRoutes#event} */
   context('event register with', function() {
-    subject(function() { return new RoutableComponentRoutes(this.routing_classes) });
+    subject(function() { return new RoutableComponentRoutes(this.routingClasses) });
     context('simple event()', function() {
-      lazy('routing_classes', function() { return MyRouting });
+      lazy('routingClasses', function() { return MyRouting });
       it('surely registered', function() {
         assert(this.subject.toString().match(/^from1.event1 => controller1#action1/));
       });
     });
     context('from()', function() {
-      lazy('routing_classes', function() { return MyRoutingWithFrom });
+      lazy('routingClasses', function() { return MyRoutingWithFrom });
       it('surely registered', function() {
         assert(this.subject.toString().match(/^with_from1.event1 => controller1#action1/));
         assert(this.subject.toString().match(/with_from2.event3 => controller3#action3\n$/));
       });
     });
     context('controller()', function() {
-      lazy('routing_classes', function() { return MyRoutingWithController });
+      lazy('routingClasses', function() { return MyRoutingWithController });
```

Comment:
__[eslint]__
* Error - 'lazy' is not defined. (no-undef)
* Warning - Too many nested callbacks (4). Maximum allowed is 3. (max-nested-callbacks)
* Warning - Unexpected function expression. (prefer-arrow-callback)
* Error - Missing semicolon. (semi)

<!-- Comment by [SideCI](https://sideci.com) -->